apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.stories.tsx

```diff
@@ -21,15 +21,11 @@ const Template: ComponentStory<typeof TrendingSuggestionList> = (args) => (
 export const Default = Template.bind({});
 Default.args = {
   isLoading: false,
-  isFetching: false,
-  isFetched: true,
```

Comment:
Can you tell me more about why these were removed?

Reply:
I removed these because in our last component `TrendingSuggestionList` we were passing `isFetched` and `isFetching` as props to our `TrendingItemSuggestion`, but I refactored it to pass isLoading right before pushing the last PR. 

After doing so I forgot to change the story, and it was causing an error which wouldn't let me continue with configuring MSW. I probably should have made the change its on PR.